replicate-to-sql-with-generic-events/scripts/delete-data.js

```diff
@@ -0,0 +1,37 @@
+const { Kuzzle, WebSocket } = require('kuzzle-sdk');
+
+const kuzzle = new Kuzzle(new WebSocket('localhost'));
+
+const indexName = 'nyc-open-data';
+const collectionName = 'yellow-taxi';
+
+async function searchData() {
+  const result = await kuzzle.document.search(indexName, collectionName, {}, { size: 500 });
+  return result.hits;
+}
+
+async function deleteData(ids = []) {
+  try {
+    const result = kuzzle.document.mDelete(indexName, collectionName, ids);
+    await kuzzle.collection.refresh(indexName, collectionName);
+    return result;
+  } catch (error) {
+    throw new Error(error);
```

Comment:
Useless wrapping of the `error` inside a new `Error` class.  
Also the `try...catch` is useless since you are just re-throwing the exception